docs/overview/style-props.md

```diff
@@ -394,7 +394,7 @@ Set the opacity of the border. Value 0, `LV_OPA_0` or `LV_OPA_TRANSP` means full
 </ul>
 
 ### border_width
```

Comment:
This is a generated so these typos needs to be fixed in [style_api_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/style_api_gen.py).